client/app/widgets/ToggleLocale/index.jsx

```diff
@@ -36,7 +37,12 @@ const onChange = (e: SyntheticEvent<HTMLInputElement>) => {
 export const ToggleLocale = (props: Props) => {
   const { locale, locales } = props;
   return (
-    <div aria-label={I18n.t('language')}>
```

Comment:
I was looking into this further and there's a simpler way of doing implementing this without having to introduce more CSS to hide elements.

We would want to extend our Input component to add a prop called hiddenLabel: boolean. If it's true then we don't want to render InputLabel. But we do want to put an aria-label on the input/select element itself.

Let me know if you want to discuss this further :)

Reply:
The reason why the CSS is that way is to make it hearable by screen readers, but not visible to mar the design. Hiding the label altogether and replacing it with an aria label is a solution, but screen reader support for ARIA is not consistent, sadly. When a label is possible, I shoot for a label.

On the note of this component, I see an error on the live website:
<img width="598" alt="screen shot 2018-10-16 at 6 53 50 am" src="https://user-images.githubusercontent.com/8865558/47025630-52955380-d110-11e8-9141-5583a1aec60e.png">

Reply:
That being said, I will definitely test the `aria-label` solution with a couple of screen readers!

Reply:
Thanks for explaining! :)

It looks like that issue you're getting on the live site has to do with our cache which I'll look into. Can you try doing a hard refresh? Thanks for the heads up.

Reply:
Whoa...
![image](https://user-images.githubusercontent.com/8865558/47049262-411c6d80-d149-11e8-9801-d242c430aafb.png)
I think #1162 targets the white text bug. 
I'm seeing this on a Windows 10 OS and Firefox 62.0.3 browser.

Reply:
"Combobox. English. Collapsed." is what NVDA, Firefox, Windows 10 says. I almost wonder if putting `aria-label` on the input, rather than the div, would help the chances of "Languages" being read aloud.

Reply:
That makes sense to me! ^